lleaves/compiler/codegen/codegen.py

```diff
@@ -317,6 +317,7 @@ def _populate_objective_func_block(
     llvm_copysign = builder.module.declare_intrinsic(
         "llvm.copysign", (DTYPE, DTYPE), ir.FunctionType(DTYPE, (DTYPE, DTYPE))
     )
+    llvm_maxnum = builder.module.declare_intrinsic("llvm.maxnum", (DTYPE, DTYPE), ir.FunctionType(DTYPE, (DTYPE, DTYPE)))
```

Comment:
Thanks, I agree with you.
However, `maximum` cannot work for me (and maybe a common case). When I try `maximum`, It turns out as "LLVM ERROR: Cannot select: t65: f64 = fmaximum t55, t64".
I tried to find out the cause, listed below. I'am not familiar with llvm, so I cannot ensure the below is correct.

1. The issue is simliar to https://github.com/llvm/llvm-project/issues/53353 . From replies, it seems that maximum of float was supported not long ago: https://github.com/llvm/llvm-project/commit/a82d27a9a6853c96f857ba0f514a78cd03bc5c35
2. For _llvmlite_, I installed the latest version 0.43.0 by pre-built binary (_pip_ or _conda_), and the required LLVM is integrated by llvmlite binary (https://llvmlite.readthedocs.io/en/latest/admin-guide/install.html). I believe this is a common usage. The integrated LLVM doesn't seems to support the float version of `maximum`.

Reply:
Huh yeah it seems this is not implement. I guess it doesn't super matter, since we're currently doing this elementwise anyway (instead of e.g. using `llvm.vector.reduce.fmax`) you could just implement it via `fcmp` followed by `select`.